### PR TITLE
add descriptions to pipelines

### PR DESCRIPTION
**Pull request by arewm**
As well as more metadata to the Tekton bundles.

# Before you complete this pull request ...

Look for any open pull requests in the repository with the title "e2e-tests update" and 
see if there are recent e2e-tests updates that will be applicable to your change.

### CODE REVIEW

File: pipelines/docker-build-multi-platform-oci-ta/patch.yaml [hunk 1]
```diff
@@ -1,4 +1,11 @@
 ---
+- op: add
+  path: /spec/description
+  value: |
+    This pipeline is ideal for building multi-arch container images from a Containerfile while maintaining trust after pipeline customization.
+
+    _Uses `buildah` to create a multi-platform container image leveraging [https://konflux-ci.dev/architecture/ADR/0036-trusted-artifacts.html](trusted artifacts). It also optionally creates a source images and runs some build-time tests. This pipeline requires that the [https://github.com/konflux-ci/multi-platform-controller](multi platform controller) is deployed and configured on your {ProductName} instance. Information is shared between tasks using OCI artifacts instead of PVCs. EC will pass the [https://enterprisecontract.dev/docs/ec-policies/release_policy.html#trusted_task__trusted](trusted_task.trusted) policy as long as all data used to build the artifact is generated from trusted tasks.
```
Comment (tnevrlka) [inline pipelines/docker-build-multi-platform-oci-ta/patch.yaml:7]: typo: "creates **a** source image**s**"

File: pipelines/fbc-builder/patch.yaml [hunk 2]
```diff
@@ -1,4 +1,11 @@
 ---
+- op: add
+  path: /spec/description
+  value: |
+    This pipeline is ideal for building and verifying xref:/advanced-how-tos/building-olm.adoc#building-the-file-based-catalog[file-based catalogs].
+
+    _Uses `buildah` to create a container image. Its build-time tests are limited to verifying the included catalog and not to scanning the image itself.
```
Comment (tnevrlka) [inline pipelines/fbc-builder/patch.yaml:7]: nit: the sentence `Its build-time tests are limited to verifying the included catalog and not to scanning the image itself.` sounds a bit unclear to me 

How about something like `Its build-time tests are limited to verifying the included catalog and do not scan the image.`?

File: pipelines/docker-build/patch.yaml [hunk 3]
```diff
@@ -1,4 +1,11 @@
 ---
+- op: add
+  path: /spec/description
+  value: |
+    This pipeline is ideal for building container images from a Containerfile while reducing network traffic.
+
+    _Uses `buildah` to create a container image. It also optionally creates a source images and runs some build-time tests. EC will flag a violation for [https://enterprisecontract.dev/docs/ec-policies/release_policy.html#trusted_task__trusted](trusted_task.trusted) if any tasks are added to the pipeline.
```
Comment (tnevrlka) [inline pipelines/docker-build/patch.yaml:7]: typo: "creates **a** source image**s**"

**Review by tnevrlka — COMMENTED** [review]
LGTM otherwise

**Review by tkdchen — APPROVED** [review]

**Pull request merged**